charts/osm/templates/prometheus-configmap.yaml

```diff
@@ -128,32 +132,32 @@ data:
           target_label: __metrics_path__
           replacement: /api/v1/nodes/${1}/proxy/metrics/cadvisor
 
-      - job_name: 'kubernetes-service-endpoints'
-        kubernetes_sd_configs:
-        - role: endpoints
-        relabel_configs:
-        - source_labels: [__meta_kubernetes_service_annotation_prometheus_io_scrape]
-          action: keep
-          regex: true
-        - source_labels: [__meta_kubernetes_service_annotation_prometheus_io_scheme]
-          action: replace
-          target_label: __scheme__
-          regex: (https?)
-        - source_labels: [__meta_kubernetes_service_annotation_prometheus_io_path]
-          action: replace
-          target_label: __metrics_path__
-          regex: (.+)
-        - source_labels: [__address__, __meta_kubernetes_service_annotation_prometheus_io_port]
-          action: replace
-          target_label: __address__
-          regex: ([^:]+)(?::\d+)?;(\d+)
-          replacement: $1:$2
-        - action: labelmap
-          regex: __meta_kubernetes_service_label_(.+)
-        - source_labels: [__meta_kubernetes_namespace]
-          action: replace
-          target_label: kubernetes_namespace
-        - source_labels: [__meta_kubernetes_service_name]
-          action: replace
-          target_label: kubernetes_name
+      # - job_name: 'kubernetes-service-endpoints'
```

Comment:
Could you drop a comment in the file why these are commented out and what are the next steps.

Reply:
I can do that. Basically I've seen we use no metrics from these endpoints, it's simple as that.
I'd like for @snehachhabria to take a look too as I'm not aware of these being used anywhere, but I'd rather have it confirmed.

Reply:
If we don't need it, might as well remove it. If we leave it here commented, let's document it in code and follow it up with an issue.